pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/UniformLoadShedder.java

```diff
@@ -144,6 +145,10 @@ public Multimap<String, String> findBundlesForUnloading(final LoadData loadData,
                             return Triple.of(bundle, bundleData, throughput);
                         }).filter(e -> !recentlyUnloadedBundles.containsKey(e.getLeft()))
                         .sorted((e1, e2) -> Double.compare(e2.getRight(), e1.getRight())).forEach((e) -> {
+                            if (conf.getMaxUnloadBundleNumPerShedding() != -1
+                                    && unloadBundleCount.getValue() >= conf.getMaxUnloadBundleNumPerShedding()) {
```

Comment:
```suggestion
                                    && selectedBundlesCache.size()  >= conf.getMaxUnloadBundleNumPerShedding()) {
```
We don't need a new variable here.

Reply:
fixed. PTAL,thanks!  @Demogorgon314